### PR TITLE
Demoapp update

### PR DESCRIPTION
**Pull request by anwzhang**
The goal is to move each combination of setup into its own activity, so that when a Publisher is looking for an example, it can quickly go into one activity without being distracted by other examples.

### CODE REVIEW

File: Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/CustomApplication.java [hunk 1]
```diff
@@ -44,8 +44,6 @@ public void onCreate() {
 
         PrebidMobile.setShareGeoLocation(true);
         PrebidMobile.setApplicationContext(getApplicationContext());
-        WebView obj = new WebView(this);
-        obj.clearCache(true);
```
Comment (yoalex5) [inline Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/CustomApplication.java:48]: why did you remove these lines ?

Comment (anwzhang) [reply Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/CustomApplication.java:48]: Why do we have this?

Comment (yoalex5) [reply Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/CustomApplication.java:48]: I don't know
You added it as a part of commit
c3b0a76b222894f7941f78cb9102cbead724137d

Usually I use this technic to review all requests and responses(no caching) using Charles

Comment (anwzhang) [reply Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/CustomApplication.java:48]: I don't remember why I added this, it seems not meaningful to the demo app.
Do you approve the removing?

**Review by yoalex5 — APPROVED** [review]
